var/spack/repos/builtin/packages/grib-util/package.py

```diff
@@ -19,7 +19,7 @@ class GribUtil(CMakePackage):
     version("develop", branch="develop")
     version("1.5.0", tag="v1.5.0", commit="b84e877a62efe6695546a4b7a02e7adb6e1ece25")
     version("1.4.0", tag="v1.4.0", commit="eeacc9ec93dfe6379f576191883c84a4a1202cc8")
-    version("1.3.0", commit="9d3c68a")
+    version("1.3.0", commit="9d3c68ab3219773141e69ee2097e45feda0e66e1")
```

Comment:
It looks like I added this commit has a year ago but it doesn't match with our v1.3.0 tag... @edwardhartnett @Hang-Lei-NOAA unless there's some issue I'm not thinking of, then @haampie can you please change the commit hash to 29d59b67724bb3c5e21d5efe7c4e903fb65a2ef4 and add `tag="v.1.3.0"`?